### DISCUSSION TITLE
Очистка плейлиста от треков, которые "правообладатель временно убрал из каталога"

### DISCUSSION THREAD

**Discussion by kisoid** [Chimildic/YaMuTools#57] [Идея] [answered]
Не знаю, как Яндекс поступит с теми лэйблами, которые не захотят или не смогут продлить лицензию на музыку, но предполагаю, что количество временно недоступных треков будет ближайшее время расти.

В связи с этим, предлагаю subj.

Будет полезно, так как иногда воспроизведение прерывается, когда доходит очередь до отключенного трека. Да и просто для наведения порядка.

**Comment by Chimildic** [comment, marked as answer]
Сомневаюсь в необходимости прерывания воспроизведения, а вот фильтр ожидаем в следующей версии расширения.

**Reply by AlexByte** [reply]
Фильтр треков-Управление недоступными
